### PR TITLE
Nanotrasen Harrier-class Exploration Cruiser

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request
harrier
harrier_areas

_Originally an advanced ICW-era command cruiser, the surviving Harriers have been refurbished as exploration and "reclamation" ships in the post-ICW era. Large, well-rounded, and boasting excellent crew accomodations, Harriers are exceedingly popular with Nanotrasen crews in spite of their lack of specialized equipment._

Replaces both the Osprey and the Skipper with the Harrier, filling roughly the same role. Key features:
- Internal Affairs office
- Security office with working brig
- Dedicated AI core
- Working mail and disposals system
- A fancy atrium
- The sector's biggest cafeteria

Hopefully this irons out some of the more common issues with the Osprey and makes for a more well-rounded and enjoyable ship that _actually gives Nanotrasen security slots for once._

## Why It's Good For The Game

Osprey is crungled and Skipper competes for niche space. Harrier should do better than either, with  more polish and faction flavor.

## Changelog

:cl:
add: Added Nanotrasen Harrier-class cruiser
del: Removed Osprey-class cruiser
del: Removed Skipper-class cruiser
/:cl:

### CODE REVIEW

File: _maps/configs/nanotrasen_harrier.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
```
Comment (Sadhorizon) [inline _maps/configs/nanotrasen_harrier.json:1]: Add `"faction": "/datum/faction/nt"`, somewhere, or it's gonna show as indie on the spawning menu.

**Comment by Thera-Pissed** [comment]
could you put an atmos analyzer in the atmos tech's locker or somewhere in that room? Techs dont get one roundstart and there's no blue toolboxes

**Comment by Apogee-dev** [comment]
alright, issues should be fixed. please can another maptainer review or just merge it man it's been five months and i'm tired

**Review by meemofcourse — APPROVED** [review]
fix conflicts and i think we can marg

**Pull request merged**